websites/G/GenoAnime/presence.ts

```diff
@@ -0,0 +1,58 @@
+const presence = new Presence({
+  clientId: "810203651317432351"
+}),
+browsingStamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+  const presenceData: PresenceData = {
+    largeImageKey: "geno",
+    details: "Genoanime !!!"
+  },
+title = document.title; //title of the page
+
+  if (document.location.pathname == "/" ) {
+    presenceData.startTimestamp = browsingStamp;
+    presenceData.details = "Exploring Genoanime";
+	presenceData.buttons = [{label:"Visit Genoanime",url: document.location.href}];
```

Comment:
![Screenshot_2021-04-04-08-58-34-75.jpg](https://user-images.githubusercontent.com/54318514/113496505-8654d580-9524-11eb-95bc-b866ab73e280.jpg)

Reply:
Completed removed redirect to home page